<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertReservation</class>
 <widget class="QDialog" name="InsertReservation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>고객정보</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>기존고객</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insertCustomerBtn">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>신규고객</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="searchData">
         <property name="placeholderText">
          <string>이름/전화번호 검색</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="searchBtn">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>검색</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="searchCustomerView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>411</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>예약정보</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2,2">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="baseSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>날자 선택</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="reservationDate"/>
     </item>
     <item>
      <widget class="QTimeEdit" name="reservationTime"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,4">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>시술금액</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="price">
       <property name="minimum">
        <number>10000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>5000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>391</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>메모</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="memo"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>예약금</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="deposit">
       <property name="minimum">
        <number>10000</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="singleStep">
        <number>5000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>상태</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="state_0">
       <property name="text">
        <string>대기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="state_1">
       <property name="text">
        <string>확정</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="state_2">
       <property name="text">
        <string>완료</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="state_3">
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="state_4">
       <property name="text">
        <string>노쇼</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>시술정보</string>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3,1">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>시술 선택</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="serviceType"/>
     </item>
     <item>
      <widget class="QCheckBox" name="retouch">
       <property name="text">
        <string>리터치</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>580</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_8">
    <item>
     <widget class="QPushButton" name="insertBtn">
      <property name="text">
       <string>등록</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="updateBtn">
      <property name="text">
       <string>수정</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>취소</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
